nvm.sh

```diff
@@ -1789,7 +1789,7 @@ nvm_get_make_jobs() {
   elif [ "_$NVM_OS" = "_sunos" ]; then
     NVM_CPU_THREADS="$(psrinfo | wc -l)"
   elif [ "_$NVM_OS" = "_aix" ]; then
-    NVM_CPU_THREADS="$(lsconf | command grep 'Number Of Processors:'| command awk '{print $4}')"
+    NVM_CPU_THREADS="$(pmcycles -m | wc -l)"
```

Comment:
I'm not sure I understand - does AIX abstract over "processors" such that programs should be using N - 1 *threads* in order to avoid locking up the machine?